From 9b7eff0c2961f5022854bce7f6f5e04177439ba6 Mon Sep 17 00:00:00 2001
From: Han Lu <hanlu@codeaurora.org>
Date: Mon, 19 Aug 2019 15:49:52 +0800
Subject: [PATCH 1/1] udev: Add rules for QTI SA8155

In SA8155, the ALSA devices enumeration is done by audio policy
manager. Bypass udev enumeration to avoid conflicting.

Signed-off-by: Han Lu <hanlu@codeaurora.org>

diff --git a/src/modules/alsa/mixer/profile-sets/90-pulseaudio.rules b/src/modules/alsa/mixer/profile-sets/90-pulseaudio.rules
index 264bfdb..0a4f678 100644
--- a/src/modules/alsa/mixer/profile-sets/90-pulseaudio.rules
+++ b/src/modules/alsa/mixer/profile-sets/90-pulseaudio.rules
@@ -22,6 +22,9 @@ SUBSYSTEMS=="usb", GOTO="pulseaudio_check_usb"
 
 SUBSYSTEMS=="platform", DRIVERS=="thinkpad_acpi", ENV{PULSE_IGNORE}="1"
 
+# QTI SA8155
+DRIVERS=="sa8155-asoc-snd", ENV{PULSE_IGNORE}="1"
+
 # Force enable speaker and internal mic for some laptops
 # This should only be necessary for kernels 3.3, 3.4 and 3.5 (as they are lacking the phantom jack kctls).
 # Acer AOA150
-- 
1.9.1

